KHANAAS-GUIDE.md

```diff
@@ -1,17 +1,38 @@
 # Guide for implementing Khan AAS
 
 ### Create a Django Project
- - `vagrant ssh -c 'django-admin startproject khanaas .'`
+ - Create a project directory `mkdir khanaas`
+ - `vagrant ssh -c 'django-admin startproject khanaas ./khanaas'`
```

Comment:
I like it better with the root directory being where the manage.py is located.  Check out http://www.kennethreitz.org/essays/repository-structure-and-python, which also addresses the topic.  Adding one more directory in the path just makes all the following commands unecessarily redundant (e.g. 'Place database setting in `./khanaas/khanaas/local_settings.py` file')